src/core/ext/filters/http/message_compress/message_compress_filter.cc

```diff
@@ -93,6 +107,8 @@ class ChannelData {
   grpc_compression_algorithm default_compression_algorithm_;
   /** Enabled compression algorithms */
   grpc_core::CompressionAlgorithmSet enabled_compression_algorithms_;
+  std::unique_ptr<grpc_core::CompressionOptions> compression_options_;
+  int default_grpc_min_message_size_to_compress_;
```

Comment:
Please call this `min_message_size_to_compress_`.

Reply:
done